style.css

```diff
@@ -1,8 +1,79 @@
-@font-face {
-  font-family: comic;
-  src: url("../fonts/ComicSansMS3.ttf");
-}
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100&family=Lato:ital,wght@0,100;0,300;0,400;0,700;1,100&family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,700;1,600&display=swap');
 
 * {
-  font-family: comic, 'Times New Roman', Times, serif !important;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  text-align: center;
+}
+
+#books_display {
+  width: 70%;
+  margin: auto;
+  display: none;
+}
+
+.bookList {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+.bookList:nth-child(odd) {
+  background: #ccc;
+}
+
+.bookInfo {
+  display: flex;
+  font-weight: 700;
+}
+
+span {
+  padding: 9px;
+}
+
+.bookInfo > div {
+  margin: 10px;
+}
+
+.border {
+  padding: 10px;
+  width: 80px;
+  border-bottom: 4px solid black;
+  margin: auto;
+  display: none;
+}
+
+#add_book {
```

Comment:
- [OPTIONAL] Please make you add button aligned to the right, as in the wireframe. Please kindly fix that. See image below
![add](https://user-images.githubusercontent.com/65852621/143233012-accc2255-7e9d-44a5-97c7-89fdb382e476.png)
.